base/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM continuumio/miniconda3:latest
+
+# XRootD + CA + VOMS + Coffea
+RUN conda install --yes --freeze-installed \
+      -c conda-forge \
+      conda-build \
+      voms \
+      python==3.8.* \
+      ca-policy-lcg \
+      xrootd==5.0.3 \
+      coffea==0.6.45 \
+      && conda clean -tipsy \
+      && find /opt/conda/ -type f,l -name '*.a' -delete \
```

Comment:
When I run containers in OkD, they are not allowed to modify the image, which means any such auto-generated files are not available. I am not sure if this is a significant performance impact or not, but it is something to consider. How much container size is saved with these deletes?

Reply:
@nsmith- hmmm...for conda it is super needed operation
```
REPOSITORY               TAG                 IMAGE ID            CREATED             SIZE                SHARED SIZE         UNIQUE SIZE         CONTAINERS
<none>                   <none>              6ed0967e714f        8 minutes ago       1.101GB             428.8MB             672.2MB             0
vs.
<none>                   <none>              dd3a0f239482        17 seconds ago      2.096GB             428.8MB             1.667GB             0

```
Basically it is about almost 1 GB of difference

Reply:
Another reason that size of the image is bigger, could be that I should add `python == 3.8` because of conflicts for `xrootd`: https://gist.github.com/oshadura/a661ba01899b90a748c5e27a2991c621

Reply:
Is that with vs. without the `find` commands? Or also removing `conda clean` (which should be ok)

Reply:
ha, you are right without 3 lines with `find` https://github.com/CoffeaTeam/docker-coffea-base/pull/1/files#diff-b093d34eec92adf6e0cc6e445edb7147ea45d25fb0c98f2f84a60f5aae6f34f7R13-R15 the size becomes much more reasonable:   ```REPOSITORY               TAG                 IMAGE ID            CREATED             SIZE                SHARED SIZE         UNIQUE SIZE         CONTAINERS
<none>                   <none>              384f2c0ef6c3        13 seconds ago      1.262GB             428.8MB             833.6MB             0``